### PR TITLE
Settings - Writing: add a settings group for Widgets

### PR DESCRIPTION
**Pull request by eliorivero**
This PR introduces a new group of settings under the Writing tab in the Settings page. The controls available in this group allow users of a Jetpack site to activate the Extra Sidebar Widgets and Widget Visibility modules.

#### Changes proposed in this Pull Request

* add new settings group titled "Widgets" for Jetpack sites
* add toggle to activate additional widgets
* add toggle to activate Widget Visibility

Captura de Pantalla 2019-04-16 a la(s) 12 15 46

Captura de Pantalla 2019-04-16 a la(s) 12 32 17

#### Testing instructions

* go to http://calypso.localhost:3000/settings/writing/YOURSITE
* verify that the toggles are there and that they look good
* turn the Widgets toggle on, and verify in https://elio.blog/wp-admin/admin.php?page=jetpack_modules&module_tag=Appearance that Extra Sidebar Widgets is on
* turn the Widgets toggle off, and verify in https://elio.blog/wp-admin/admin.php?page=jetpack_modules&module_tag=Appearance that Extra Sidebar Widgets is off
* turn the Widget Visibility toggle on, and verify in https://elio.blog/wp-admin/admin.php?page=jetpack_modules&module_tag=Appearance that Widget Visibility is on
* turn the Widget Visibility toggle off, and verify in https://elio.blog/wp-admin/admin.php?page=jetpack_modules&module_tag=Appearance that Widget Visibility is off

Part of https://github.com/Automattic/jetpack/issues/11933 and https://github.com/Automattic/jetpack/issues/11936

### CODE REVIEW

File: client/my-sites/site-settings/widgets.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		return (
+			<FormFieldset>
+				<SupportInfo
+					text={ translate( 'Configure widgets to appear only on certain posts or pages.' ) }
+					link="https://jetpack.com/support/widget-visibility"
+				/>
+
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="widgets"
```
Comment (MichaelArestad) [inline client/my-sites/site-settings/widgets.jsx]: This should be `moduleSlug="widget-visibility"`

File: client/my-sites/site-settings/widgets.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+		return (
+			<FormFieldset>
+				<SupportInfo
+					text={ translate( 'Provides additional widgets for use on your site.' ) }
+					link="https://jetpack.com/support/extra-sidebar-widgets/"
+				/>
+
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="widget-visibility"
```
Comment (MichaelArestad) [inline client/my-sites/site-settings/widgets.jsx]: This should be `moduleSlug="widgets"`

File: client/my-sites/site-settings/widgets.jsx (hunk 1 above)
Comment (eliorivero) [reply client/my-sites/site-settings/widgets.jsx]: Oh no! Fixing it.

File: client/my-sites/site-settings/widgets.jsx (hunk 2 above)
Comment (eliorivero) [reply client/my-sites/site-settings/widgets.jsx]: All done. Sorry.

**Comment by MichaelArestad** [comment]
Works properly, now. Thanks!

**Pull request merged**